packages/gcf-utils/test/gcf-bootstrapper.ts

```diff
@@ -342,24 +384,202 @@ describe('GCFBootstrapper', () => {
       sinon.assert.calledOnce(configStub);
       sinon.assert.notCalled(sendStatusStub);
       sinon.assert.calledOnceWithMatch(sendStub, {statusCode: 500});
-      sinon.assert.notCalled(spy);
+      sinon.assert.notCalled(issueSpy);
+      sinon.assert.notCalled(repositoryCronSpy);
+      sinon.assert.notCalled(installationCronSpy);
+      sinon.assert.notCalled(globalCronSpy);
       downloaded.done();
     });
 
-    it('ensures that task is enqueued when called by scheduler for many repos', async () => {
-      await mockBootstrapper();
-      req.body = {
-        installation: {id: 1},
-      };
-      req.headers = {};
-      req.headers['x-github-event'] = 'schedule.repository';
-      req.headers['x-github-delivery'] = '123';
-      req.headers['x-cloudtasks-taskname'] = '';
-      nockListInstallationRepos();
+    describe('per-repository cron', () => {
+      it('ensures that task is enqueued when called by scheduler for many repos', async () => {
+        await mockBootstrapper();
+        req.body = {
+          installation: {id: 1},
+        };
+        req.headers = {};
+        req.headers['x-github-event'] = 'schedule.repository';
+        req.headers['x-github-delivery'] = '123';
+        req.headers['x-cloudtasks-taskname'] = '';
+        nockListInstallationRepos();
```

Comment:
Normally we call `.done()` for nock scopes. Do you want to do the same?

Reply:
Good call, it turns out some of the nock scopes were unnecessary.